### PR TITLE
Some utility data for oncoprint in resultsviewpagestore

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Comment by alisman** [comment]
Should this be going to release-1.10.0?

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 1]
```diff
@@ -297,33 +295,216 @@ export class ResultsViewPageStore {
...
-                return filterCBioPortalWebServiceData(this.oqlQuery, mutations, (new accessors(this.selectedMolecularProfiles.result!)), this.defaultOQLQuery.result!, undefined, true);
+    readonly caseAggregatedData = remoteData<CaseAggregatedData<AlterationData>>({
+        await: ()=>[
+            this.filteredAlterations,
+            this.samples,
+            this.patients
+        ],
+        invoke: ()=>{
+            return Promise.resolve({
+                samples:
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts]: something kind weird about formatting here

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStore.ts]: its for line length

**Comment by adamabeshouse** [comment]
@alisman it should be going to whichever release oncoprint is targeted at. would that be 1.11?

**Comment by adamabeshouse** [comment]
Going to put all this in a branch off oncoprint

**Pull request closed without merging**